src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+
+        var driveSettings = outerLayout.getLayout("Drive Subsystem", BuiltInLayouts.kList).withPosition(0, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _flOffset = driveSettings.add("FL Offset", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frOffset = driveSettings.add("FR Offset", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blOffset = driveSettings.add("BL Offset", 0.0).withPosition(0, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brOffset = driveSettings.add("BR Offset", 0.0).withPosition(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var IntakeSetting = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _IntakeInSpeed  = IntakeSetting.add("Intake In Speed", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 0).getEntry();
+        _IntakeOutSpeed = IntakeSetting.add("Intake Out Speed", 0.0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 1).getEntry();
+
+        var climbSetting = outerLayout.getLayout("Climber", BuiltInLayouts.kList).withPosition(2, 0).withProperties(Map.of("label position", "LEFT")).withSize(7, 10);
+        _climberleftOffset  = climbSetting.add("Left Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberRightOffset = climbSetting.add("Right Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMinHeight   = climbSetting.add("Arm Minimum Height", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMaxHeight   = climbSetting.add("Arm Maximum Height", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var noteSetting = outerLayout.getLayout("Notepath", BuiltInLayouts.kList).withPosition(3, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _NoteShootSpeed     = noteSetting.add("Note Shoot Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _PickupIntakeSpeed  = noteSetting.add("Pickup Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterIntakeSpeed = noteSetting.add("Shooter Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var shooterBedSetting = outerLayout.getLayout("Shooter Bed", BuiltInLayouts.kList).withPosition(4, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _bedMinimumAngle = shooterBedSetting.add("Bed Minimum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _bedMaximumAngle = shooterBedSetting.add("Bed Maximum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterOffset   = shooterBedSetting.add("Shooter Relative Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
1. Updated tab names
2. Updated variable names
3. Use constants for defaults
4. Remove `withSize`
5. Add entries for bed angle when picking up through intake and through shooter:

```suggestion
        var shooterBedSettings = outerLayout.getLayout("Shooter Bed", BuiltInLayouts.kList).withPosition(4, 0).withProperties(Map.of("label position", "LEFT"));
        _bedMinimumAngle       = shooterBedSettings.add("Bed Minimum Angle", Constants.ShooterBed.MIN_BED_ANGLE.getDegrees()).withWidget(BuiltInWidgets.kTextView).getEntry();
        _bedMaximumAngle       = shooterBedSettings.add("Bed Maximum Angle", Constants.ShooterBed.MAX_BED_ANGLE.getDegrees()).withWidget(BuiltInWidgets.kTextView).getEntry();
        _shooterOffset         = shooterBedSettings.add("Shooter Offset", Constants.ShooterBed.BED_ANGLE_OFFSET.getDegrees()).withWidget(BuiltInWidgets.kTextView).getEntry();
        _bedPickupIntakeAngle  = shooterBedSettings.add("Bed Pickup Intake Angle", Constants.ShooterBed.BED_INTAKE_PICKUP_ANGLE.getDegrees()).withWidget(BuiltInWidgets.kTextView).getEntry();
        _bedPickupShooterAngle = shooterBedSettings.add("Bed Pickup Shooter Angle", Constants.ShooterBed.BED_SHOOTER_PICKUP_ANGLE.getDegrees()).withWidget(BuiltInWidgets.kTextView).getEntry();
```